packages/client/test/integration/StreamrClient.test.ts

```diff
@@ -150,7 +150,7 @@ describe('StreamrClient', () => {
         })
     })
 
-    describe('utf-8 encoding', () => {
```

Comment:
I suppose UTF-8 encoding should work in the browser as well. We don't validate it in the browser tests now at all since the input in this test and `integration/Resends2.test.ts` is read from a file. I suppose  the file is useful since the content covers a wide array of characters.

Reply:
Both tests read the file. I made it so that in the browser we use a hardcoded string instead of reading the file: 930233bf20d95463234bbbea84277e024f8cecb0